packages/graphql-transformers-e2e-tests/src/__tests__/DynamoDBModelTransformer.e2e.test.ts

```diff
@@ -9,11 +9,14 @@ import AppSyncTransformer from 'graphql-appsync-transformer'
 import { CloudFormationClient } from '../CloudFormationClient'
 import { Output } from 'aws-sdk/clients/cloudformation'
 import { GraphQLClient } from '../GraphQLClient'
+import * as moment from 'moment';
```

Comment:
Did we previously depend on moment? If not there needs to be an update to package.json devDependencies for all these packages.

Reply:
I added moment dependency for ES tests that got merged like a couple of days ago.